python/cudf/cudf/_lib/cpp/io/types.pxd

```diff
@@ -32,6 +32,10 @@ cdef extern from "cudf/io/types.hpp" \
         BROTLI "cudf::io::compression_type::BROTLI"
         ZIP "cudf::io::compression_type::ZIP"
         XZ "cudf::io::compression_type::XZ"
+        ZLIB "cudf::io::compression_type::ZLIB"
+        LZ4 "cudf::io::compression_type::LZ4"
+        LZO "cudf::io::compression_type::LZO"
+        ZSTD "cudf::io::compression_type::ZSTD"
```

Comment:
These were simply missing

Reply:
Were these accidentally missing? If so, do we need to update our docs saying we have end-user support for these compression types?

Reply:
I added these values to match libcudf's `compression_type` at
https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/include/cudf/io/types.hpp#L55
From the docs above it seemed like this is the intent. Let me know if this is not the case.

Adding ZLIB to the `to_orc` docs.